packages/vue/src/components/alias-control.vue

```diff
@@ -11,14 +11,14 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, computed } from "vue";
-import { authInputAttributes } from "@aws-amplify/ui-core";
+import { defineComponent, computed } from 'vue';
```

Comment:
This happened because of prettier resolving merge conflicts.

Reply:
Yeah that's fine